test/runtests.jl

```diff
@@ -3,20 +3,20 @@ using Test
 using UUIDs
 
 const DEPOT = joinpath(@__DIR__, "resources")
-const FOOBAR_REGISTRY = reachable_registries("Foobar"; depots=DEPOT)
+const FOOBAR_REGISTRY = first(reachable_registries("Foobar"; depots=DEPOT))
 
 
 @testset "internal functions" begin
     @testset "_get_pkg_name" begin
         @testset "uuid to name" begin
             expected = "Case1"
-            pkg_name = PkgDeps._get_pkg_name(UUID("00000000-1111-2222-3333-444444444444"); registries=[FOOBAR_REGISTRY])
+            pkg_name = PkgDeps._get_pkg_name(UUID("00000000-1111-2222-3333-444444444444"); depots=DEPOT)
```

Comment:
why this change?

Is this the fix mentioned?

> Fixed bug with users() would not allow you to look for usages of a package outside of pkg_registry. For example, if I wanted to look up all usages of DataFrames in an internal repository, I could not unless DataFrames was registered in the internal repository.

I think we could really do with a test for the "look up all usages of DataFrames in an internal repository" (something that failed before this PR but works in this one)

Reply:
I did this in an attempt to simplify testing. The signature,

```julia
_get_pkg_name(::UUID; kwargs...)
```

Just wraps around

```julia
_get_pkg_name(::UUID, ::RegistryInstance[])
```

This cuts from 2 tests to 1 test